src/runtime_src/core/pcie/emulation/cpu_em/generic_pcie_hal2/shim.h

```diff
@@ -253,6 +254,7 @@ namespace xclcpuemhal2 {
       exec_core* mCore;
       SWScheduler* mSWSch;
       bool mIsKdsSwEmu;
+      xclBin *mHeader;
```

Comment:
Not needed.

Reply:
Removed xclbin *mHeader and used mCoreDevice to get IP_LAYOUT section.